sway/extensions.c

```diff
@@ -157,10 +167,6 @@ static void set_lock_surface(struct wl_client *client, struct wl_resource *resou
 		desktop_shell.is_locked = true;
 		input_init();
 		arrange_windows(workspace, -1, -1);
-		swayc_t *focus_output = swayc_active_output();
-		if (focus_output == output) {
-			set_focused_container(view);
-		}
```

Comment:
Yeah, it worked when I tested it. I think it's because the keypresses are sent to the frontmost wlc view?